### DISCUSSION TITLE
На что менять blynk?

### DISCUSSION THREAD

**Discussion by dontsovcmc** [dontsovcmc/waterius#195] [General]
Насколько я вижу поддержка старой бесплатной версии прекратилась. Новая версия приложения требует абонентской платы. На что можно заменить (кроме написания своего приложения) - предлагайте!

**Comment by heX16** [comment]
Есть еще [ifttt.com](https://ifttt.com/).
Но вроде там бесплатного тарифа хватит максимум для одной лампочки...
